### DISCUSSION TITLE
About launching external apps

### DISCUSSION THREAD

**Discussion by darkj2k** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#58] [Q&A] [answered]
This feature is so great but I'm not a programmer so hope someone can help me. I did studied the examples but still has no clue. I like to using this function for bypass {searchTerms} to Directory Opus for searching local files. Following is the command:

`dopusrt.exe /cmd Find {searchTerms} IN F:\Movies`

How do I make it as a bookmarklet? Thanks in advance.

**Comment by YoshifumiFuyuno** [comment, marked as answer]
Hi.

I apologize for the lack of explanation.
I will enhance the examples a bit more when I have more time.
Well, I have tried it with the Directory Opus Pro 12.29 Build 8272 trial version.
It seems to work below.

<a href="#SPC%7B%22name%22%3A%22Directory%20Opus%22%2C%22searchUrl%22%3A%22javascript%3Aspc.spc.sendNativeMessage(%7Bpath%3A%20String.raw%60%5C%22C%3A%5C%5CProgram%20Files%5C%5CGPSoftware%5C%5CDirectory%20Opus%5C%5Cdopusrt.exe%5C%22%20%2Fcmd%20Find%20%5C%22%24%7B%7BsearchTerms%7D%7D%5C%22%20IN%20%5C%22F%3A%5C%5CMovies%5C%22%60%7D)%3B%22%2C%22iconUrl%22%3A%22https%3A%2F%2Fwww.gpsoft.com.au%2Ffavicon.ico%22%2C%22encode%22%3A%22RAW%22%7D"><img src="https://www.gpsoft.com.au/favicon.ico" width="16"><span class="engineName">Directory Opus</span></a>

source code:
```javascript
javascript:spc.spc.sendNativeMessage({path: String.raw`"C:\Program Files\GPSoftware\Directory Opus\dopusrt.exe" /cmd Find "${{searchTerms}}" IN "F:\Movies"`});
```
icon: https://www.gpsoft.com.au/favicon.ico

Best regards

**Comment by darkj2k** [comment]
Amazing. It's what I need and it works. Thanks sooo much.
